src/cs/LionWeb.Integration.WebSocket.Tests/Client/ContainmentClientTests.cs

```diff
@@ -80,12 +250,38 @@ public void MoveChildFromOtherContainmentInSameParent()
 
         AssertEquals(aPartition, bPartition);
 
-        bPartition.Containment_1 = bPartition.Containment_0_1;
+        bPartition.Containment_1 = bPartition.Containment_0_1!;
         aClient.WaitForReplies(1);
 
         AssertEquals(aPartition, bPartition);
     }
+    
+    /// <summary>
+    /// Moves a child from one containment to another within the same parent and replaces the existing child node, if any.
```

Comment:
"if any" should not happen in a test. Either it's there and we replace it, or not.